sw/host/rom_ext_image_tools/signer/src/main.rs

```diff
@@ -40,16 +40,18 @@ fn main() {
     // Update "signed" manifest fields.
     image.update_static_fields(&config);
 
-    let exponent = &signature_key_public_exponent_le();
-    image.update_exponent_field(exponent);
-
-    let modulus = &signature_key_modulus_le();
-    image.update_modulus_field(modulus);
-
     // Convert ASN.1 DER private key into Mundane RsaPrivKey.
     let private_key =
         RsaPrivKey::parse_from_der(&private_key_der).expect("Failed to parse private key!");
 
```

Comment:
Maybe add a comment like:

```rust
// Encode public key components in little-endian byte order.
```

Just to hint why the `reverse()` is needed.

Reply:
Done.